src/test/ui/proc-macro/proc-macro-deprecated-attr.rs

```diff
@@ -0,0 +1,15 @@
+// build-pass
+
```

Comment:
Could you add `#![deny(deprecated)]` here to make sure that the test fails if the deprecation is actually reported.